### DISCUSSION TITLE
Is it possible to use +experiment=path/experiment_name?

### DISCUSSION THREAD

**Discussion by zhiyuanpeng** [ashleve/lightning-hydra-template#192] [Q&A] [answered]
Hi, I have many experiment files under `config/experiments`, and I want to put the `experiment.yaml` under `configs/experiment/exp_name/`  so that I can group the experiments by `exp_name`. How to run experiments from `configs/experiment/exp_name/`? Thanks a lot.

**Comment by ashleve** [comment, marked as answer]
@zhiyuanpeng try something like this
```
python run.py experiment=group1/exp1
python run.py experiment=group1/exp2
python run.py experiment=group2/exp3
```
(groups are just extra folders in `configs/experiment/`)
